galsim/wfirst/wfirst_detectors.py

```diff
@@ -82,7 +82,28 @@ def applyIPC(img, edge_treatment='extend', fill_value=None):
     """
     img.applyIPC(galsim.wfirst.ipc_kernel, edge_treatment=edge_treatment, fill_value=fill_value)
 
-def allDetectorEffects(img, rng=None, exptime=None):
+def applyPersistence(img, prev_exposures):
+    """
+    Applies the persistence effect to the Image instance by adding a small fraction of the previous
+    exposures (up to 8) supplied as the 'prev_exposures' argument.
```

Comment:
Yes, the pixel values in the previous exposures should correspond to the electron counts, since it is the electrons that persist. I have included this in the docstring. Also, that made me realize that the `allDetectorEffects` must return the updated list of previous exposures since any user using this routine does not have access to the image that needs to be queued.